javascript/ql/lib/semmle/javascript/NPM.qll

```diff
@@ -340,8 +340,8 @@ class PackageDependencies extends JsonObject {
     )
   }
 
-  /** Holds if this package depends on version 'version' of package 'pkg'. */
-  predicate getADependency(string pkg, string version) { version = this.getPropStringValue(pkg) }
+  /** Returns the version of the specified package that this package depends on. */
+  string getADependency(string pkg) { result = this.getPropStringValue(pkg) }
```

Comment:
I know this is in draft PR state, but I a peek anyway.

We have to be wary of breaking changes in predicates that are part of the public API. To change the signature of this predicate, we'd have to rename it and leave behind a deprecated version of the original predicate.

Apart from that, I think I'd prefer to rename it to `hasDependency` instead, and not add a return value.

Reply:
And if we rename it to `hasDependency`, we would still need to keep the original predicate `getADependency` marked as deprecated and in the comment refer that people now should use `hasDependency`, correct?

Reply:
Yep, that's right